catalogs/catalog-hive/src/test/java/com/datastrato/gravitino/catalog/hive/integration/test/CatalogHiveIT.java

```diff
@@ -1357,4 +1360,45 @@ public void testPurgeHiveExternalTable() throws TException, InterruptedException
     Assertions.assertTrue(
         hdfs.listStatus(tableDirectory).length > 0, "The table should not be empty");
   }
+
+  @Test
+  @EnabledIf("isDeployMode")
```

Comment:
Why do we only enable it in deploy mode?

Reply:
Couldn't run under embedded mode, because Iceberg package may not build yet.

Reply:
Can we make the ut depend on task.jar?

Reply:
This comment is not addressed. And the way you did is tricky, you need to fix it not bypass it. @FANNG1

Reply:
Seems there is no strong necessary to make it run in embedded mode. The main purpose of this test is to load a custom catalog operations from external class path, run in deploy mode is enough.

Reply:
no need to load custom catalog operation from external class path, remove the limit now.